Simplenote/Classes/SPNoteListViewController+Extensions.swift

```diff
@@ -622,6 +697,38 @@ extension SPNoteListViewController {
             notesListController.object(at: indexPath) as? Note
         }
     }
+
+    open override func setEditing(_ editing: Bool, animated: Bool) {
+        // tableView could be in edit mode already if a single cell is displaying a slide out menu
+        // Check if tableView and NoteListVC have the same isEditing value
+        // If not set them to the same before continuing
+        if tableView.isEditing != isEditing {
+            tableView.setEditing(isEditing, animated: true)
+        }
+
+        super.setEditing(editing, animated: animated)
+        tableView.setEditing(editing, animated: animated)
+        refreshEditButtonTitle()
+        refreshSelectAllLabels()
+
+        tableView.deselectAllRows(inSection: .zero, animated: false)
+        updateNavigationBar()
+
+        configureNavigationToolbarButton()
+        setListViewTitle()
+    }
+
+
+    private func setListViewTitle() {
```

Comment:
```suggestion
    private func refreshListViewTitle() {
```

Since `setListViewTitle` implies the method is a **setter** (and you would expect a parameter!)